src/KubernetesClient/KubernetesClient.csproj

```diff
@@ -8,37 +8,41 @@
     <PackageProjectUrl>https://github.com/kubernetes-client/csharp</PackageProjectUrl>
     <PackageIconUrl>https://raw.githubusercontent.com/kubernetes/kubernetes/master/logo/logo.png</PackageIconUrl>
     <PackageTags>kubernetes;docker;containers;</PackageTags>
-
-    <LangVersion>7.3</LangVersion>
-    <TargetFrameworks>netstandard2.0;net452;netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461;netcoreapp2.1</TargetFrameworks>
```

Comment:
The entire informers is built on System.Reactive which is netstandard2.0. net461 is the earliest framework that supports netstandard2.0 spec. 

I flagged this in PR description at the top. I actually think that instead of having a single nuget package we should modularize it. Informers can be something like KubernetesClient.Informers. The controllers same thing as it built on top of informers. This also goes towards needing to have a place for `.AddKubernetes` DI method which is currently part of sample project, but that's not where it belongs - should be part of the SDK.